server/webapp/WEB-INF/rails.new/webpack/views/dashboard/material_changes_widget.js.msx

```diff
@@ -27,6 +28,18 @@ const MaterialRevisionsWidget = {
     };
   },
 
+  onupdate(vnode) {
+    this.adjustPositionOfPopup = () => {
+      const widthOfWindow            = window.innerWidth;
+      const rightCoordinateOfWidget  = vnode.dom.getBoundingClientRect().right;
+      if (rightCoordinateOfWidget > widthOfWindow) {
+        $(vnode.dom).css({right: 0, position: 'absolute'});
```

Comment:
Can we just change this to add the appropriate `style` attribute instead of `jQuery.css`